app/controllers/albums.js

```diff
@@ -1,6 +1,6 @@
 const albumsService = require('../services/albums');
 const albumsSerializer = require('../serializers/albums');
-const albumPhotosSerializer = require('../serializers/album_photos');
+const albumPhotosSerializer = require('../serializers/album-photos');
```

Comment:
Why do you change this?

Reply:
I researched online and read that they should be kebab-case just like npm packages do. However I see now that snake_case is the convention according to Wolox' best practices.
Reverted in latest commit!